source/common/http/http1/codec_impl.h

```diff
@@ -192,7 +194,7 @@ class ConnectionImpl : public virtual Connection, protected Logger::Loggable<Log
 
   CodecStats& stats() { return stats_; }
 
-  const bool enable_trailers_;
+  const bool enable_trailers_ : 1;
```

Comment:
This needs to be packed next to the other bitfield items or there are no savings.